lib/ddtrace/contrib/rails/action_controller/callbacks/rails50.rb

```diff
@@ -0,0 +1,27 @@
+module Datadog
+  module Contrib
+    module Rails
+      module ActionController
+        module Callbacks
+          # Callbacks module specifically for Rails 5.0
```

Comment:
The comment above, applies to this one too. If the API changes, we should not break users applications. I guess we don't have other ways to plug into the controller callbacks.

Reply:
Same feedback above, too.